submodules/faxbox/faxbox.js

```diff
@@ -233,6 +261,14 @@ define(function(require) {
 
 				delete results.current_user;
 
+				var invalidCallerID = self.getCallerID(results);
+
+				if (invalidCallerID) {
```

Comment:
Since `lodash#find` returns `undefined` if no match exists, you can specifically check against that here.
```suggestion
				if (_.isUndefined(invalidCallerID)) {
```
And simply unshift the `caller_id` directly instead of an object on the next line.